helix-view/src/editor.rs

```diff
@@ -273,6 +273,11 @@ pub struct Config {
     pub terminal: Option<TerminalConfig>,
     /// Column numbers at which to draw the rulers. Defaults to `[]`, meaning no rulers.
     pub rulers: Vec<u16>,
+
+    /// If set, use this character to draw the editor's rulers.
+    #[serde(rename = "ruler-char")]
```

Comment:
This is unnecessary because the struct is tagged with `#[serde(rename_all = "kebab-case")]`